Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/entity/Certificate.java

```diff
@@ -358,7 +358,7 @@ private void loadCertFromData() throws CertificateException {
                 setThumbprint("");
             } else {
                 setThumbprint(Thumbprint.toThumbprint(cert).toString());
-                setPrivateKey(container != null && container.getKey() != null);
+                setPrivateKey(container.getKey() != null);
```

Comment:
Container cannot be null here because cert is not null in this block, and cert is populated from container.